mods/ra/chrome/ingame-leavemap.yaml

```diff
@@ -124,6 +131,13 @@ Container@LEAVE_MAP_WIDGET:
 					Height: 25
 					Font: Bold
 					Text: Leave
+				Button@RESTART_BUTTON:
+					X: PARENT_RIGHT - 3 * (WIDTH + 20)
```

Comment:
`X: 20`, please. It's looks better when the button is all the way to the left side